module/Finna/src/Finna/View/Helper/Root/Record.php

```diff
@@ -196,6 +203,20 @@ public function commentingEnabled()
                 && $this->config->Social->comments !== 'disabled');
     }
 
+    /**
+     * Is repository library request form enabled for this record.
+     *
+     * @return boolean
+     */
+    public function repositoryLibraryRequestEnabled() : bool
+    {
+        return (bool)($this->config->Record->repository_library_request ?? false)
```

Comment:
repository_library_request -asetus voisi olla pilkulla eroteltu lista data sourceista, joille lomake on käytössä. Ei tarvitsisi kovakoodata 'vaari' tai listaa ylipäänsä.

Reply:
Näin olisi yksinkertaisempi, mutta ajateltiin että olisi hyvä varmistaa että asetusta ei yliajeta.